src/main/java/graphql/language/AstPrinter.java

```diff
@@ -575,28 +575,28 @@ String wrap(String start, String maybeString, String end) {
             }
             return "";
         }
-        return start + maybeString + (!isEmpty(end) ? end : "");
+        return new StringBuilder().append(start).append(maybeString).append(!isEmpty(end) ? end : "").toString();
```

Comment:
This actually makes a difference in the benchmarks - not a lot but some

Reply:
This may no longer be true when *compiling* with JDK 9
https://openjdk.java.net/jeps/280

I do not know how long you want to stay on JDK 8 though, for now it is an improvement